myriad-scheduler/src/main/java/com/ebay/myriad/api/ClustersResource.java

```diff
@@ -60,38 +60,89 @@ public ClustersResource(MyriadConfiguration cfg,
     @Timed
     @PUT
     @Path("/flexup")
-    @Produces(MediaType.APPLICATION_JSON)
+    @Produces(MediaType.TEXT_PLAIN)
     @Consumes(MediaType.APPLICATION_JSON)
     public Response flexUp(FlexUpClusterRequest request) {
         Preconditions.checkNotNull(request,
                 "request object cannot be null or empty");
 
-        // TODO(mohit): Validation
         LOGGER.info("Received Flexup Cluster Request");
 
-
         Integer instances = request.getInstances();
         String profile = request.getProfile();
 
-        LOGGER.info("Instances: ", instances);
-        LOGGER.info("Profile: ", profile);
+        LOGGER.info("Instances: {}", instances);
+        LOGGER.info("Profile: {}", profile);
+
+        // Validate profile request
+        Response.ResponseBuilder response = Response.status(Response.Status.ACCEPTED);
+        if (!this.profileManager.exists(profile)) {
+            response.status(Response.Status.BAD_REQUEST)
+                    .entity("Profile does not exist: " + profile);
+            LOGGER.error("Provided profile does not exist " + profile);
```

Comment:
nit - make the profile name passed in appear more distinctly in the log.error. for e.g. use single quotes around the variable 'profile'.